src/extension.ts

```diff
@@ -78,11 +82,16 @@ import {
 import { createTestController } from './testController';
 import {
   type LegendConceptTreeItem,
+  type LegendConceptTreeProvider,
   createLegendConceptTreeProvider,
 } from './conceptTree';
 import { renderDiagramRendererWebView } from './webviews/DiagramWebView';
+import { renderServiceQueryEditorWebView } from './webviews/ServiceQueryEditorWebView';
 
 let client: LegendLanguageClient;
+let legendConceptTreeProvider: LegendConceptTreeProvider;
+
+const queryBuilderPanels: Map<string, WebviewPanel> = new Map();
```

Comment:
added a similar variable in main branch for Diagrams. Please use anyone.

Reply:
Done